src/app/campaign-details/campaign-details.component.html

```diff
@@ -0,0 +1,23 @@
+<img class="banner" [src]="campaign.bannerUri" alt="Campaign Banner">
+<div class="details container">
+    <div>
+        <h2>{{campaign.title}} <em *ngIf="campaign.championName"> | Championed by: {{campaign.championName}}</em></h2>
+        <h3 *ngIf="campaign.isMatched">Matched Campaign!</h3>
+        <h4 *ngIf="!campaign.isMatched">Regular campaign</h4>
+    </div>
+
+    <div>
+        <h3>Campaign Summary</h3>
+        <p>{{campaign.summary}}</p>
+    </div>
+
+    <div>
+        <h3>Campaign Targets</h3>
+        <p *ngIf="campaign.target > 0">Campaign Target: {{ campaign.target | currency : '£' : 'symbol' : '1.0-0' }}</p>
+        <p *ngIf="campaign.amountRaised > 0">Amount Raised: {{ campaign.amountRaised | currency : '£' : 'symbol' : '1.0-0' }}</p>
+    </div>
+
+    <div>
+        <button mat-raised-button color="primary" [routerLink]="'/donate/' + campaign.id">Donate</button>
+    </div>
+</div>
```

Comment:
It's normal practice to have 1 newline at the end of files to keep diffs cleaner while following OS conventions - this is why GitHub adds this marker if it's missing